org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/groovy/SmartApp.groovy

```diff
@@ -102,25 +102,28 @@ def installed() {
 
 def updated() {
 	log.debug "Updating with settings: ${settings}"
-	if(state.deviceSubscriptionMap == null){
+	
+    //Initialize state variables if didn't exist.
+    if( state.deviceSubscriptionMap == null ){
 		state.deviceSubscriptionMap = [:]
-		log.debug "deviceSubscriptionMap created."
```

Comment:
It is nice to know people are reading them. I will put them back  :)

---
In reply to: [112324268](https://github.com/openT2T/translators/pull/234#discussion_r112324268) [](ancestors = 112324268)